omero/sysadmins/unix/walkthrough/walkthrough_ubuntu1604.sh

```diff
@@ -19,10 +19,17 @@ apt-get install -y openjdk-11-jre
 apt-get update
 apt-get -y install \
 	unzip \
-	wget \
-	python3 \
-	python3-venv
+	wget
+apt-get -y install python3 python3-venv
 #end-step01
+#start-step01-py36
+add-apt-repository ppa:deadsnakes/ppa
```

Comment:
Not to cast aspersions on their effort, but do we know anything about https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa ?

Reply:
we could build from source instead